src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -1184,9 +1244,427 @@ End Interface
+        {
+            var testText = @"
+using System;
+namespace PushUpTest
+{
+    interface IInterface
+    {
+    }
+
+    public class TestClass : IInterface
+    {
+        private event EventHandler Event1, Eve[||]nt2, Event3;
+    }
+}";
+
+            var expected = @"
+using System;
+namespace PushUpTest
+{
+    interface IInterface
+    {
+        event EventHandler Event1;
+        event EventHandler Event2;
+        event EventHandler Event3;
+    }
+
+    public class TestClass : IInterface
+    {
+        public event EventHandler Event1;
+        public event EventHandler Event2;
+        public event EventHandler Event3;
+    }
+}";
+            await TestWithPullMemberDialogAsync(testText, expected);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsPullMemberUp)]
+        public async Task PullMethodToInnerInterfaceViaDialog()
+        {
+            var testText = @"
+using System;
+namespace PushUpTest
+{
+    public class TestClass : TestClass.IInterface
+    {
+        private void Bar[||]Bar()
+        {
+        }
+        interface IInterface
+        {
+        }
+    }
+}";
+            var expected = @"
+using System;
+namespace PushUpTest
+{
+    public class TestClass : TestClass.IInterface
+    {
+        public void BarBar()
+        {
+        }
+        interface IInterface
+        {
+            void BarBar();
+        }
+    }
+}";
+
+            await TestWithPullMemberDialogAsync(testText, expected);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsPullMemberUp)]
+        public async Task PullDifferentMembersFromClassToPartialInterfaceViaDialog()
+        {
+            var testText = @"
+using System;
+namespace PushUpTest
+{
+    partial interface IInterface
+    {
+    }
+
+    public class TestClass : IInterface
+    {
+        public int th[||]is[int i]
+        {
+            get => j = value;
+        }
+
+        private static void BarBar()
+        {}
+        
+        protected static event EventHandler event1, event2;
+
+        internal static int Foo
+        {
+            get; set;
+        }
+    }
+    partial interface IInterface
+    {
+    }
+}";
+
+    var expected = @"
+using System;
+namespace PushUpTest
+{
+    partial interface IInterface
+    {
+        int this[int i] { get; }
+
+        int Foo { get; set; }
+
+        event EventHandler event1;
+        event EventHandler event2;
+
+        void BarBar();
+    }
+
+    public class TestClass : IInterface
+    {
+        public int this[int i]
+        {
+            get => j = value;
+        }
+
+        public void BarBar()
+        {}
+
+        public event EventHandler event1;
+        public event EventHandler event2;
+
+        public int Foo
+        {
+            get; set;
+        }
+    }
+    partial interface IInterface
+    {
+    }
+}";
+            await TestWithPullMemberDialogAsync(testText, expected, index : 1);
```

Comment:
```suggestion
            await TestWithPullMemberDialogAsync(testText, expected, index: 1);
```

Reply:
Oh, How could you find such typo mistakes.😅

Reply:
I'm actually surprised the IDE let this through.  it seems like formatting-analyzer woudl have caught this.  @sharwell ?